pages/resources/index.js

```diff
@@ -156,12 +177,60 @@ export const getStaticProps = async ({ params }) => {
     return a.kicker.localeCompare(b.kicker);
   });
 
+  // compile page content based on kicker. Includes a name, the posts to be displayed, a description, and the size of the posts
+  // we have to hard-code What's New because it's too unique
+  let content = { categories: [] };
+  let [name, posts, description, size, type, id] = [
+    "What's New?",
+    recents,
+    null,
+    "full",
+    "New",
+    "What's%20New?",
+  ];
+  let category = { name, posts, description, size, type, id };
+  content.categories.push(category);
+
+  let [label, level, headings] = [name, 2, []];
+  let heading = { label, level };
+  headings.push(heading);
+
+  const descriptions = {
+    Guides:
+      "Explore the processes and tools we use in our step-by-step written guides.",
+    Tutorials:
+      "Watch or read through our tutorials to understand key techniques and concepts.",
+    Workshops:
+      "Sharpen your design and development skills with our in-depth recorded workshops.",
+  };
+
+  for (let i = 0; i < collections.length; i++) {
+    name = collections[i].kicker;
+    if (name === "Guides") {
+      posts = [...collections[i].docs]
+        .sort((a, b) => a.data.guideRank - b.data.guideRank)
```

Comment:
What if the data doesn't have a guideRank? How are you handling the errors?

Reply:
No error is thrown and everything behaves correctly because Chrome handles invalid arguments by resolving to sort them alphanumerically! But I can still add error handling in case other browsers don't support this :-)